merlin-server/constraints-dsl-compiler/src/libs/constraints-ast.ts

```diff
@@ -162,6 +164,13 @@ export interface RealProfileGreaterThanOrEqual {
   right: RealProfileExpression;
 }
 
+export interface RealProfileIsWithin {
+  kind: NodeKind.RealProfileIsWithin;
+  left: RealProfileExpression;
+  right: RealProfileExpression;
+  margin: RealProfileExpression;
+}
+
```

Comment:
For edsl-only wrappers like this, you don't need to create a new AST node. AST changes are only needed when creating a new node in the Java backend.